adafruit_framebuf.py

```diff
@@ -215,6 +215,32 @@ def vline(self, x, y, height, color):
         """Draw a vertical line up to a given length."""
         self.rect(x, y, 1, height, color, fill=True)
 
+    def circle(self, center_x, center_y, radius, color):
+        """Draw a circle at the given midpoint location, radius and color. The ```circle``` method draws only
```

Comment:
It looks like this line is too long. Simple fix. Just take some of the text off the end and add it in front of the text on the line below.